src/cljs/velho_ds/molecules/field.cljs

```diff
@@ -80,26 +81,43 @@
 ;; INPUTS
 (defn input-field [{:keys [label content placeholder icon-click-fn on-change-fn on-blur-fn on-focus-fn styles]}]
```

Comment:
These fixes affect dropdown-menu too. Need to check it works with them too before merge